README.md

```diff
@@ -1,3 +1,3 @@
 Elf Attack
-
-Now with slightly less ReadMe.
\ No newline at end of file
+This is another version that i will not merge immediately,
+so we can see if Sheila can approve it.
```

Comment:
Is there somewhere we can "approve" the PR or would we just leave a comment to that effect?